RPIPs/RPIP-4.md

```diff
@@ -14,65 +14,68 @@ The Rocket Pool community at large agrees to use Snapshot voting for governance
 The Snapshot voting mechanism uses staked and weighted RPL token voting.
 
 ## Motivation
-Governance in Rocket Pool is currently ad hoc, and there is no way for the community to adopt proposals. We must adopt a governance mechanism, even if imperfect, before any governance actions can be taken legitimately, such as but not limited to spending pDAO funds.
+Governance in Rocket Pool is currently ad hoc, and there is no way for the community to adopt proposals. We must adopt a governance mechanism, even if imperfect, before any governance actions can be taken legitimately, such as spending pDAO funds.
 
 ## Implementation
   
-All Rocket Pool governance proposals must first be posted as a new topic on the governance forums at dao.rocketpool.net. Topics should include a poll with the following format:
+All Rocket Pool governance proposals MUST have an associated post on the governance forums at dao.rocketpool.net.
+Promising community sentiment and a corresponding RPIP document are REQUIRED for snapshot voting to be scheduled.
+
+Topics SHOULD approximate community sentiment by including a poll with the following format which is live for at least [7] days:
 
 ```
 [poll type=regular results=always chartType=bar]
 * Support
 * Oppose
+* Third option
+* etc
 [/poll]
 ```
 
-A favorable forum poll which is live for at least [7] days and a corresponding RPIP document are necessary for snapshot voting to be scheduled. Once the requirements have been met, one of the following parties will create a snapshot vote and publicize it:
-
-- Darren Langley AKA langers (General Manager - Rocket Pool Pty Ltd)
-- Mike Leach AKA Wander (RPIP Editor, Founder - Bamboo Financial Technologies)
-- [others?]
+A favorable forum poll which is live for at least [7] days and a corresponding RPIP document are necessary for snapshot voting to be scheduled. Once the requirements have been met, either 1) Darren Langley AKA langers (General Manager - Rocket Pool Pty Ltd) or 2) a party he entrusts with this ability SHOULD create a snapshot vote and publicize it as soon as reasonably possible unless exercizing their veto power (see Vetoing below). 
```

Comment:
The first sentence here overlaps with the above. I'd suggest just starting with "Once the requirements..."

Reply:
Fixed